cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,137 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+  cy.visit('/');
+});
+
+it('Pagination on the page', () => {
+  cy.contains('.-pagination', 'Previous')
+    .should('exist');
+  cy.contains('.-pagination', 'Next')
+    .should('exist');
+  cy.contains('.-pagination', 'Page')
+    .should('exist');
+  });
+
+it('Should have the ability to add worker', () => {
+  cy.contains('.btn', 'Add')
+    .click();
+  cy.get('#firstName')
+    .type('Rosa');
+  cy.get('#lastName')
+    .type('Winters');
+  cy.get('#userEmail')
+    .type('rowisa@yep.com');
+  cy.get('#age')
+    .type('18');
+  cy.get('#salary')
+    .type('5600');
+  cy.get('#department')
+    .type('Cepka');
+  cy.get('#submit')
+    .click();
+
+  cy.contains('.rt-td', 'Rosa', 'Winters', '18', '5600', 'Cepka')
+    .should('exist');
+});
+
+it('Should have the ability to change rows count selection', () => {
+  cy.get('[aria-label="rows per page"]').select('10')
+    .should('contain', '10 rows');
+  });
+
+it('Should have the ability to delete worker', () => {
+  cy.get('#delete-record-3')
+    .click();
+
+  cy.get('#delete-record-3')
+    .should('not.exist');
+  });
+
+it('Should have the ability to delete all workers', () => {
+  cy.get('#delete-record-1')
+    .click();
+  cy.get('#delete-record-2')
+    .click();
+  cy.get('#delete-record-3')
+    .click();
+
+  cy.contains('.rt-noData', 'No rows found')
+    .should('exist');
+  });
+
+it('Should have the ability to search a worker, edit it and validate data in the worker row', () => {
+  cy.get('#searchBox')
+    .type('Cierra');
+  cy.get('#basic-addon2')
+    .click();
+  cy.get('#edit-record-1')
+    .click();
+  cy.get('#firstName')
+    .clear()
+    .type('Zozulya');
+  cy.get('#lastName')
+    .clear()
+    .type('Gorobchyk');
+  cy.get('#userEmail')
+    .clear()
+    .type('gorobchyk@example.com');
+  cy.get('#age')
+    .clear()
+    .type('40');
+  cy.get('#salary')
+    .clear()
+    .type('11000');
+  cy.get('#department')
+    .clear()
+    .type('Engineer');
+  cy.get('#submit')
+    .click();
+
+  cy.contains('.rt-tr -odd', 'Zozulya', 'Gorobchyk', 'gorobchyk@example.com', '40', '11000', 'Engineer')
```

Comment:
The function `.contains()` requires two arguments, string and text to match. Here you are trying to pass multiple string values which may cause unintended behavior. Consider using multiple `.contains()` assertions for different values.